bundler.d/assets.rb

```diff
@@ -15,4 +15,8 @@
 gem 'sass-rails', '~> 5.0'
 gem 'spice-html5-rails', '~> 0.1.5'
 gem 'quiet_assets', '~> 1.0'
+# Pinned until https://github.com/reactjs/react-rails/issues/431 is resolved
+gem 'react-rails', '1.4.1'
```

Comment:
These 3 are unpackaged so I guess we have to package them, unless you know of a repo with these.